jsonnet/custom.libsonnet

```diff
@@ -79,6 +79,21 @@
               severity: 'critical',
             },
           },
+          {
+            alert: 'highOverallControlPlaneIOWait',
+            expr: |||
+              (sum(irate(node_cpu_seconds_total {mode="iowait"} [2m])) without (cpu)) / count(node_cpu_seconds_total) without (cpu) * 100 > 20
+              AND on (instance) label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
+            |||,
+            'for': '5m',
```

Comment:
for a warning level alert, we can wait for 1h before firing the alert

Reply:
i fixed this on the PR against api-operator